tools/run_command.py

```diff
@@ -27,12 +29,19 @@ def run_build_examples(argv):
 def run_print_plugins(arv):
     print_plugins.run_print_plugins(arv)
 
+# Analyze plugin packages
+
+
+def run_analyze_plugins(arv):
+    analyze_plugins.run_analyze_plugins(arv)
+
 
 commands = {}
 commands['tidy'] = {'func': run_check_tidy, 'info': 'Check and update format for C++ files'}
 commands['test'] = {'func': run_integration_test, 'info': 'Run integration test'}
 commands['build'] = {'func': run_build_examples, 'info': 'Build examples of plugin'}
 commands['plugins'] = {'func': run_print_plugins, 'info': 'Print plugins list'}
+commands['analyze'] = {'func': run_analyze_plugins, 'info': 'Analyze plugins for Dart files'}
```

Comment:
How about "Analyze the plugin's Dart code" for consistency with `flutter-tizen analyze`?

```suggestion
commands['analyze'] = {'func': run_analyze_plugins, 'info': 'Analyze the plugin's Dart code'}
```

Reply:
I applied this, thank you.